debug/debug-wordpress.md

```diff
@@ -4,6 +4,8 @@ Debugging PHP code is part of any project, but WordPress comes with specific deb
 
 For non-programmers or general users, these options can be used to show detailed information about errors.
 
+**NOTE**: Prior to making any modifications to your website, verify that you have either utilized a staging environment or taken an appropriate backup of your site.
+
 ## Example wp-config.php for Debugging
 
 The following code, inserted in your [wp-config.php](https://wordpress.org/documentation/article/editing-wp-config-php/) file, will log all errors, notices, and warnings to a file called `debug.log` in the wp-content directory. It will also hide the errors so they do not interrupt page generation.
```

Comment:
The new URL for “wp-config” is:
https://developer.wordpress.org/advanced-administration/wordpress/wp-config/